File: SheCollectionBE/SheCollectionBE/DB Models/Booking.cs
using System;
using System.Collections.Generic;

namespace SheCollectionBE.Models
{
    public partial class Booking
    {
        public int BookingId { get; set; }
        public int? CustomerId { get; set; }
        public int? EmployeeId { get; set; }
        public int? ScheduleId { get; set; }
        public int? TimeslotId { get; set; }
        public int? BookingStatusId { get; set; }
        public string BookingDescription { get; set; } = null!;

        public virtual BookingStatus? BookingStatus { get; set; }
        public virtual CustomerTable? Customer { get; set; }
        public virtual Employee? Employee { get; set; }
        public virtual Schedule? Schedule { get; set; }
        public virtual Timeslot? Timeslot { get; set; }
        public virtual ServiceTable ServiceTable { get; set; }
    }
}

File: FrontEnd/src/app/models/booking/booking.ts
import { Service } from '../services/Service';
import { TimeSlot } from '../services/TimeSlot';
import { Customer } from '../user/Customer';
import { Schedule } from '../user/Schedule';
import { staffMember } from '../user/StaffMember';

export interface Booking {
  bookingId: number;
  bookingDescription: string;
  bookingStatus: any;
  customer: Customer;
  employee: staffMember;
  schedule: Schedule;
  timeslot: TimeSlot;
  serviceTable: Service;
}
